plugin/cr/src/test/resources/ersd-bundle-example.json

```diff
@@ -108,16 +108,6 @@
 				},
-				"extension": [
-					{
-						"url": "http://hl7.org/fhir/StructureDefinition/variable",
-						"valueExpression": {
-							"name": "normalReportingDuration",
-							"language": "text/fhirpath",
-							"expression": "14"
-						}
-					}
```

Comment:
This extension is only valid for PlanDefinitions in r5 not r4

Reply:
Normal Reporting Duration is an important part of the workflow - see Parameter "E" here: https://build.fhir.org/ig/HL7/case-reporting/electronic_reporting_and_surveillance_distribution_ersd_transaction_and_profiles.html - and it will definitely be present in the specifications we'll be authoring. So, we should leave this in the test and then work out the solution (e.g., an R4 extension).

Reply:
The variable extension is defined (status: draft) in R4: https://hl7.org/fhir/R4/extension-variable.html

Reply:
<img width="306" alt="image" src="https://github.com/cqframework/cqf-ruler/assets/18551992/e785605a-bbd5-4ea5-a8e7-1aece9aed39d">

It's still only defined for Questionnaires 🙈 so won't help us

Reply:
reverted the change 👍🏾